webpack.config.js

```diff
@@ -98,19 +100,14 @@ module.exports = (env, argv) => {
             ]
         },
         plugins: [
-            new ModuleFederationPlugin({
+
+            new ModuleFederationPlugin(getModuleFederationConfig(packageJson, {
                 name: "jahiaUi",
                 library: { type: "assign", name: "appShell.remotes.jahiaUi" },
-                filename: "remoteEntry.js",
                 exposes: {
                     '.': './src/javascript/shared',
                     './init': './src/javascript/init',
```

Comment:
You could remove this line, the init is already exposed by the getModuleFederationConfig